### PR TITLE
bears/python: Add Flake8Bear

### PR DESCRIPTION
**Pull request by srisankethu**
Fixes https://github.com/coala/coala-bears/issues/169

### CODE REVIEW

File: bears/python/Flake8Bear.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                     r'(?P<message>.*)',
+        severity_map={
+            "F": RESULT_SEVERITY.MAJOR,
+            "E": RESULT_SEVERITY.MAJOR,
+            "W": RESULT_SEVERITY.NORMAL,
+            "N8": RESULT_SEVERITY.INFO,
+            "C9": RESULT_SEVERITY.INFO})
+class Flake8Bear:
+    """
+    Checks for styling errors based on pep8, pyflakes, pycodestyle and mccabe
```
Comment (Adrianzatreanu) [inline bears/python/Flake8Bear.py]: hey, first of all, this needs a period.

File: bears/python/Flake8Bear.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    """
+
```
Comment (Adrianzatreanu) [inline bears/python/Flake8Bear.py]: could you please fill in the metadata attributes, such as author, requirements, etc? You can check any other bear to see what you have to write :)

File: bears/python/Flake8Bear.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    @staticmethod
+    def create_arguments(filename, file, config_file):
```
Comment (Adrianzatreanu) [inline bears/python/Flake8Bear.py]: does this bear have absolutely no arguments?

**Comment by Adrianzatreanu** [comment]
also, shouldnt you add the package to requirements.txt?

File: bears/python/Flake8Bear.py (hunk 3 above)
Comment (srisankethu) [inline bears/python/Flake8Bear.py]: It does! http://flake8.pycqa.org/en/latest/user/options.html Can you please comment what options would be helpful?

File: bears/python/Flake8Bear.py (hunk 1 above)
Comment (srisankethu) [inline bears/python/Flake8Bear.py]: I did not get you

File: bears/python/Flake8Bear.py (hunk 2 above)
Comment (srisankethu) [inline bears/python/Flake8Bear.py]: Oh! I will fill them up. I was worrying why the tests are failing and totally forgot about it.

**Comment by srisankethu** [comment]
Will add it

**Pull request closed without merging**